### ISSUE TITLE
FragmentBuilder with only Optional Args will get Null Exception

### ISSUE THREAD

**Issue by Gautham** [sockeqwe/fragmentargs#30]
If I set only optional attributes for FragmentBuilder, Ill get an exception at runtime because of the null-check. The null-check should be added only if there is atleast one non-optional argument specified.

On a side note, very useful project. Thanks for this :)

**Comment by sockeqwe** [comment]
I see, thanks for reporting!

**Comment by AndreasBoehm** [comment]
I will try to create a PR for this because this bug is really annoying

**Comment by sockeqwe** [comment]
@AndreasBoehm PR are very welcome. Otherwise I have planned to fix this during weekend amongst other things like native kotlin support etc.

**Comment by sockeqwe** [comment]
I'm currently working on that, but I can't remember what exactly the issue was.
For instance:

```java
 public class OnlyOptionalFragments extends Fragment {
  
  @Arg(required = false) int foo;

  @Override public void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    FragmentArgs.inject(this);
  }
}
```

Then, if I use `OnlyOptionalFragment fragment = new OnlyOptionalFragmentBuilder().build()` it works as expected. So this issue only occurs if you instantiate the fragment directly **without** the builder `OnlyOptionalFragment fragment = new OnlyOptionalFragment()`, right? Or am I missing something else?

**Comment by AndreasBoehm** [comment]
Exactly, the problem only exists when you instantiate the fragment without the builder

**Comment by sockeqwe** [comment]
All right, thanks! I will fix it now. Btw. I recommend to always use the builder see #32 to ensure that you get compile time errors if (one day) you will change the arguments of a certain fragment.

**Comment by AndreasBoehm** [comment]
Is there an ETA for releasing 3.0?

**Comment by sockeqwe** [comment]
The only thing that is missing right now is support for kotlin, which I will hopefully implement tonight (after work). After having implemented this, i will release 3.0. So probably tomorrow it will be available in maven central. You can expect 3.0 be available by Monday at the latest
